bin/semcos

```diff
@@ -542,6 +542,22 @@ make DESTDIR=${SEMCOS_DESTDIR}/ install
 
 cd ${SEMCOS_DESTDIR}/builds
 
+# ----------- SQLITE -----------
+# This will eventually be installed through Quantum
+CFLAGS_ORIG=CFLAGS
+CFLAGS="-DSQLITE_ENABLE_DBSTAT_VTAB -DSQLITE_ENABLE_COLUMN_METADATA \
+ -DSQLITE_ENABLE_UNLOCK_NOTIFY -DSQLITE_SECURE_DELETE -DSQLITE_ENABLE_JSON1 \
+ -DSQLITE_ENABLE_FTS3 -DSQLITE_ENABLE_FTS3_PARENTHESIS -DSQLITE_ENABLE_FTS4 \
+ -DSQLITE_ENABLE_FTS3_TOKENIZER=1 -DSQLITE_ENABLE_BATCH_ATOMIC_WRITE=1 \
+ -DSQLITE_ENABLE_DESERIALIZE"
+ CFLAGS+=" -DHAVE_FDATASYNC"
```

Comment:
This is fine, make it better. CFLAS_ORIG, why. I'm OK with having extra flags in CFLAS if it's related to sqlite only. Also, move this up so everything gets impacted by it, so the compiled code is consistent.